File: hotel-server/hotel-common/src/main/java/com/xunni/hotel/entity/Booking.java
package com.xunni.hotel.entity;

import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

@Data
@TableName("orders")
public class Booking {
    @TableId
    private Integer orderId;
    private String orderNo;
    private String roomNumber;
    private String guestName;
    private String guestPhone;
    private String checkInDate;
    private String checkOutDate;
    private Integer nights;
    private Float totalAmount;
    private String orderStatus;
    private String paymentStatus;
    private String bookChannel;

}

File: hotel-web/src/types/booking.ts
export interface Booking {
  orderNo: string
  roomId: string
  roomNumber: string
  guestName: string
  guestPhone: string
  idCard: string
  checkInDate: string
  checkOutDate: string
  nights: number
  adults: number
  children: number
  totalAmount: number
  paidAmount: number
  orderStatus: string
  paymentStatus: string
  paymentMethod: string
  bookChannel: string
  checkInType: string
  customerType: string
  remarks: string
}
